### PR TITLE
Document CTAS functionality for transactional Hive tables

### PR DESCRIPTION
**Pull request by MiguelWeezardo**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -749,3 +749,14 @@ Hive 3 Related Limitations
...
+* :doc:`/sql/create-table-as` can be used to create transactional tables in ORC format like this::
+
+    CREATE TABLE blah
+    WITH (
+        format='ORC',
+        transactional=true,
+    )
+    AS SELECT <some sql query>
+
+Table statistics are currently not gathered during CTAS.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/hive.rst:761]: These need to be indented two spaces to match the list item

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -749,3 +749,14 @@ Hive 3 Related Limitations
...
+To collect those ``ANALYZE TABLE COMPUTE STATISTICS`` needs to be issued *on Hive* after table is created.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/hive.rst:762]: Make this
```
To collect statistics, run ``ANALYZE TABLE COMPUTE STATISTICS`` *using Hive* after the table is created.

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 3]
```diff
@@ -749,3 +749,14 @@ Hive 3 Related Limitations
   not correctly read ``timestamp`` values from Parquet, RCBinary, or Avro
   file formats created by Hive 3.1 or later. When reading from these file formats,
   Presto returns different results than Hive.
+* :doc:`/sql/create-table-as` can be used to create transactional tables in ORC format like this::
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/hive.rst:752]: Add newlines between all of the list items for readability (now that we have a list item with blank lines)

**Comment by MiguelWeezardo** [comment]
@electrum @losipiuk Could one of you please merge this? I'm still not able to, my merge button is disabled.

**Review by losipiuk — APPROVED** [review]

**Comment by losipiuk** [comment]
It LGTM but I would prefer someone who actually knows English to read it :) 
@electrum @mosabua ?

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 4]
```diff
@@ -741,11 +741,25 @@ Hive 3 Related Limitations
...
+
+      CREATE TABLE <name>
+      WITH (
+          format='ORC',
+          transactional=true,
+      )
+      AS <query>
+
+  Note: Presto currently does not support gathering table statistics for Hive transactional tables.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/hive.rst:764]: User actual note syntax or a make it an actual sentence.

Also remove "currently"

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 5]
```diff
@@ -741,11 +741,25 @@ Hive 3 Related Limitations
...
+  To collect statistics, run ``ANALYZE TABLE COMPUTE STATISTICS`` *using Hive* after the table is created.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/hive.rst:765]:
```
You need to use Hive to gather table statistics with ``ANALYZE TABLE COMPUTE STATISTICS`` after table creation.
```

**Comment by electrum** [comment]
Thanks!

**Pull request merged**